SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -613,6 +619,10 @@ void ResetSchemaGeneratorState()
 {
 	ActorClassPathToSchema.Empty();
 	SubobjectClassPathToSchema.Empty();
+	SchemaComponentTypeToComponentSet.Empty();
+	for (ESchemaComponentType type = ESchemaComponentType::SCHEMA_Begin; type < ESchemaComponentType::SCHEMA_Count; type = (ESchemaComponentType)(type + 1)) {
```

Comment:
You can use `ForAllSchemaComponentTypes` here to do this loop.

```
ForAllSchemaComponentTypes([&](ESchemaComponentType Type)
{
    // do stuff
}
```

Reply:
thanks! done